extras/scripts/qbittorrent-port-update.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+# Description: This script updates the peer-port for the qBittorrent torrent client using its WebUI API.
+# Note: For this to work, "Bypass authentication for clients on localhost" should be enabled
+# in the WebUI settings (json key bypass_local_auth).
+
+build_default_url() {
+    port="${1:-$WEBUI_PORT}"
+    echo "http://127.0.0.1:${port}/api"
+}
+
+# default values
+WEBUI_PORT="8080"
+DEFAULT_URL=$(build_default_url)
+WGET_OPTS="--retry-connrefused --tries=5"
+
+usage() {
+    echo "Usage: $0 [OPTIONS]"
+    echo ""
+    echo "Update qBittorrent peer-port via API"
+    echo ""
+    echo "Options:"
+    echo "  -h, --help             Show this help message and exit."
+    echo "  -u, --user USER        Specify the qBittorrent username."
+    echo "                         (Omit if not required)"
+    echo "  -p, --pass PASS        Specify the qBittorrent password."
+    echo "                         (Omit if not required)"
+    echo "  -P, --port PORT        Specify the qBittorrent peer-port."
```

Comment:
this should accept a comma separated list of ports from Gluetun's `{{PORTS}}` to simplify usage of this script.
And then it should use `$(echo {{PORTS}} | cut -d, -f1)` to take only the first port just in case, since qbitorrent supports only one port.

Reply:
Not sure if I got it right, does it mean we should use it like this `bittorrent-port-update.sh --port {{PORTS}}`?

E.g. via `VPN_PORT_FORWARDING_UP_COMMAND=/bin/sh -c "/scripts/qbittorrent-port-update.sh --port {{PORTS}} --webui-port 9081"`

If so, it's already supported, this is exactly how I used it since beggining.

Reply:
Correct, but {{PORTS}} can have multiple ports so for convenience and a shorter command, let's cut and use the first port within the script. The flag should be renamed to --ports and indicate that despite the name it only picks the first port

Reply:
I think the other way around, let's keep it named as `--port` to indicate that only one of them is used.

The logic to split and keep the first was already there
https://github.com/astappiev/gluetun/blob/839e318c9c30676c8ecd9f042c524e87a1634f32/extras/scripts/qbittorrent-port-update.sh#L60

Reply:
I updated the example to use `--port {{PORT}}`, but left `PORT=$(echo "$2" | cut -d',' -f1)` in to keep it more foolproof